cli/src/test/java/org/jboss/gm/cli/MainTest.java

```diff
@@ -129,7 +132,7 @@ public void testInvokeGroovy() throws Exception {
         m.run(args);
 
         assertTrue(systemOutRule.getLog().contains("Running Groovy script on"));
-        assertTrue(systemOutRule.getLog().contains("Tasks runnable from root project"));
+        assertTrue(systemOutRule.getLog().contains("asks runnable from root project"));
```

Comment:
Out of curiosity - did prior versions not capitalize?

Reply:
I believe that was it. I had wanted to use something like `.matches("[Tt]")` instead of `.contains()`, but out of curiosity, do we need these first two lines at all? The next three are what actually test out output, whereas these two are from Gradle itself.

Reply:
134 looks like ours, but yes you are right 135 might not be needed

Reply:
May just be better to remove this one line then.

Reply:
But isn't this line verifying that Gradle has been invoked correctly?

Reply:
Huh, good question!

Reply:
Right now I'm removed the `"asks"` line. Let me know if I should put it back. If we do, using `assertThat().matches()` may be clearer.